browser/net/brave_site_hacks_network_delegate_helper.cc

```diff
@@ -36,6 +36,8 @@ const std::string& GetQueryStringTrackers() {
            "fbclid", "gclid", "msclkid", "mc_eid",
            // https://github.com/brave/brave-browser/issues/9879
            "dclid",
+           // https://github.com/brave/brave-browser/issues/11579
+           "_openstat",
```

Comment:
wdyt about moving the whole list to the component updater?

Reply:
That's a good idea and I have an issue open for that (https://github.com/brave/brave-browser/issues/10188). The thing that needs to happen first is figuring out the right format for it.